### ISSUE TITLE
Unable to login with python 2.6

### ISSUE THREAD

**Issue by lindehoff** [persandstrom/python-verisure#18]
My home automation system is using python 2.6, the old version, before BeautifulSoup, worked fine.
But now I get the following error, I don't know how much work it will be to make it work with python 2.6.

```/Library/Python/2.6/site-packages/requests-2.8.1-py2.6.egg/requests/packages/urllib3/util/ssl_.py:100: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
/Library/Python/2.6/site-packages/requests-2.8.1-py2.6.egg/requests/packages/urllib3/connection.py:264: SubjectAltNameWarning: Certificate for mypages.verisure.com has no `subjectAltName`, falling back to check for a `commonName` for now. This feature is being removed by major browsers and deprecated by RFC 2818. (See https://github.com/shazow/urllib3/issues/497 for details.)
  SubjectAltNameWarning
/Library/Python/2.6/site-packages/bs4/builder/_htmlparser.py:167: RuntimeWarning: Python's built-in HTMLParser cannot parse the given document. This is not a bug in Beautiful Soup. The best solution is to install an external parser (lxml or html5lib), and use Beautiful Soup with that parser. See http://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser for help.
  "Python's built-in HTMLParser cannot parse the given document. This is not a bug in Beautiful Soup. The best solution is to install an external parser (lxml or html5lib), and use Beautiful Soup with that parser. See http://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser for help."))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "verisure/mypages.py", line 46, in login
    self._session.login()
  File "verisure/session.py", line 85, in login
    self.csrf = self._get_csrf()
  File "verisure/session.py", line 147, in _get_csrf
    soup = BeautifulSoup(response.text, 'html.parser')
  File "/Library/Python/2.6/site-packages/bs4/__init__.py", line 215, in __init__
    self._feed()
  File "/Library/Python/2.6/site-packages/bs4/__init__.py", line 239, in _feed
    self.builder.feed(self.markup)
  File "/Library/Python/2.6/site-packages/bs4/builder/_htmlparser.py", line 168, in feed
    raise e
HTMLParser.HTMLParseError: malformed start tag, at line 2609, column 109 ```

**Comment by persandstrom** [comment]
Ok. I'll take a look at it. It would be possible to stop using beautifulsoup if it causes problems. 

The error message sais:
Python's built-in HTMLParser cannot parse the given document. This is not a bug in Beautiful Soup. The best solution is to install an external parser (lxml or html5lib), and use Beautiful Soup with that parser. See http://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser for help.

Is that a possibility for you in the meantime?

**Comment by lindehoff** [comment]
Yes if lxml or html5lib works with Python 2.6 that's a possibly

**Comment by lindehoff** [comment]
It worked, I installed html5lib and six, then I changed the row https://github.com/persandstrom/python-verisure/blob/master/verisure/session.py#L147 to `soup = BeautifulSoup(response.text, 'html5lib')`

**Comment by persandstrom** [comment]
I reverted to regex again.
Could you please see if the branch 'remove_beautifulsoup' works better?

**Comment by lindehoff** [comment]
I have check and it seams to work fine, thx.

**Comment by persandstrom** [comment]
Fixed in #19
